src/app/views/panels/widgets/fluentui_manager.tsx

```diff
@@ -116,6 +117,7 @@ import { FluentUIGradientPicker } from "../../../components/fluent_ui_gradient_p
 import { OrderMode } from "../../../../core/specification/types";
 import { ReorderStringsValue } from "./controls/reorder_string_value";
```

Comment:
Remove import for `ReorderStringsValue`, never used.

Reply:
Fixed, thanks!